README.md

```diff
@@ -70,6 +82,66 @@ exports.config = {
     }
 };
 ```
+## Settings fot the multi threaded launch
```

Comment:
use '#### Multithreaded mode' header for this section. Move here the explanations of the problems.